vtex/faststore
# Feat: Adds Newsletter Section from CMS

## What's the purpose of this pull request?

Adjusts Newsletter and BannerNewsletter section for CMS.

## How it works?

- [x] Adjust sections in sections.json
- [x] Adapt Newsletter and BannerNewsletter to receive CMS props.
- [x] Remove Privacy Policy Disclaimer. (I don't know if it's possible to make it dynamic, since we have a link with the string)

## How to test it?

- Publish BannerNewsletter and Newsletter in CMS.
- `yarn dev` inside core package.
- Check the CMS preview.
